virtual-machine/copy-managed-disks-vhd-to-storage-account/copy-managed-disks-vhd-to-storage-account.ps1

```diff
@@ -42,6 +42,8 @@ $destinationContext = New-AzStorageContext -StorageAccountName $storageAccountNa
 if($useAzCopy -eq 1)
 {
     $containerSASURI = New-AzStorageContainerSASToken -Context $destinationContext -ExpiryTime(get-date).AddSeconds($sasExpiryDuration) -FullUri -Name $storageContainerName -Permission rw
+    $containername,$sastokenkey = $containerSASURI -split "\?"
+    $containerSASURI = "$containername/$destinationVHDFileName`?$sastokenkey"
     .\azcopy copy $sas.AccessSAS $containerSASURI
```

Comment:
This command does not run correctly under PowerShell 6.1 on Linux:

```
.\azcopy : The term '.\azcopy' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At /mnt/c/Users/sttramer/src/azure-docs-powershell-samples/virtual-machine/copy-managed-disks-vhd-to-storage-account/copy-managed-disks-vhd-to-storage-account.ps1:44 char:5
+     .\azcopy copy $sas.AccessSAS $containerSASURI
+     ~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (.\azcopy:String) [], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException
```

This is because it (wrongly) assumes that `azcopy` will always be installed in the path where the script is run from, not generally available. This issue can be fixed by removing the `.\` at the beginning of the line.